### DISCUSSION TITLE
之前升级lantern版本后突然变成了免费版？

### DISCUSSION THREAD

**Discussion by frankvaliant** [getlantern/lantern-forum-cn#291] [Q&A]
之前升级lantern版本后突然变成了免费版，然后重新换了邮箱开通了新的账号，想把原来的续到新账号上要怎么做？

**Comment by MaxMKMax** [comment]
@frankvaliant 您好. 请问您可以试看重新登陆吗?

同时请您提供我账号邮箱和购买单据. 我帮您问看.

**Reply by frankvaliant** [reply]
chengwei0324@hhdd.com，购买凭证回头我再找找看，不一定能找到了，就是今年3月13号前后，主要还剩差不多一个月了。
